corehq/apps/reports/management/commands/project_stats_report.py

```diff
@@ -192,25 +192,27 @@ def _cases_updated_per_user_per_month(self):
         self._print_table(['Month', 'Cases updated per user'], final_stats)
 
     def _ledgers_per_case(self):
-        results = (
-            LedgerES(es_instance_alias=ES_EXPORT_INSTANCE)
-            .domain(self.domain).aggregation(
-                TermsAggregation('by_case', 'case_id', size=100)
-            ).size(0).run()
-        )
+        if should_use_sql_backend(self.domain):
+            db_name = get_db_aliases_for_partitioned_query()[0]  # just query one shard DB
+            results = (
+                LedgerValue.objects.using(db_name).filter(domain=self.domain)
+                .values('case_id')
+                .annotate(ledger_count=Count('pk'))
+            )[:100]
+        else:
+            raise NotImplementedError
```

Comment:
Comment says this command is for cluster planning model. I doubted we'd run this on any couch domains except for rec (and I think that migration's underway?)